### PR TITLE
style(components): [Tabs] Modify tabs to flex layout

### LINKED ISSUE #12028: [Style] [tabs] el-tabs的添加标签页操作按钮图标颜色不对
Bug Type: **`Style`**

## Environment

- Vue Version: `3.3.0-alpha.4`
- Element Plus Version: `2.3.0`
- Browser / OS: `Chrome/111.0.0.0`
- Build Tool: `Vite`

### Related Component

- `el-tabs`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZWwtdGFic1xuICAgIHYtbW9kZWw9XCJlZGl0YWJsZVRhYnNWYWx1ZVwiXG4gICAgdHlwZT1cImNhcmRcIlxuICAgIGVkaXRhYmxlXG4gICAgY2xhc3M9XCJkZW1vLXRhYnNcIlxuICAgIEBlZGl0PVwiaGFuZGxlVGFic0VkaXRcIlxuICA+XG4gICAgIDxlbC10YWItcGFuZSBrZXk9XCJhZGRcIiBuYW1lPVwiYWRkXCI+XG4gICAgICAgPHRlbXBsYXRlICNsYWJlbD5cbiAgICAgICArICAgICAgICAgXG4gICAgICAgPC90ZW1wbGF0ZT5cbiAgICAgIDwvZWwtdGFiLXBhbmU+XG4gICAgICAgIDxlbC10YWItcGFuZSBrZXk9XCJhZGRcIiBuYW1lPVwiYWRkXCI+XG4gICAgICAgPHRlbXBsYXRlICNsYWJlbD5cbiA8RWRpdCBzdHlsZT1cIndpZHRoOiAxZW07IGhlaWdodDogMWVtOyBtYXJnaW4tcmlnaHQ6IDhweFwiIC8+XG4gICAgICAgPC90ZW1wbGF0ZT5cbiAgICAgIDwvZWwtdGFiLXBhbmU+XG4gICAgPGVsLXRhYi1wYW5lXG4gICAgICB2LWZvcj1cIml0ZW0gaW4gZWRpdGFibGVUYWJzXCJcbiAgICAgIDprZXk9XCJpdGVtLm5hbWVcIlxuICAgICAgOmxhYmVsPVwiaXRlbS50aXRsZVwiXG4gICAgICA6bmFtZT1cIml0ZW0ubmFtZVwiXG4gICAgPlxuICAgICAge3sgaXRlbS5jb250ZW50IH19XG4gICAgPC9lbC10YWItcGFuZT5cbiAgPC9lbC10YWJzPlxuPC90ZW1wbGF0ZT5cbjxzY3JpcHQgbGFuZz1cInRzXCIgc2V0dXA+XG5pbXBvcnQgeyByZWYgfSBmcm9tICd2dWUnXG5pbXBvcnQgeyBFZGl0IH0gZnJvbSAnQGVsZW1lbnQtcGx1cy9pY29ucy12dWUnXG5sZXQgdGFiSW5kZXggPSAyXG5jb25zdCBlZGl0YWJsZVRhYnNWYWx1ZSA9IHJlZignMicpXG5jb25zdCBlZGl0YWJsZVRhYnMgPSByZWYoW1xuICB7XG4gICAgdGl0bGU6ICdUYWIgMScsXG4gICAgbmFtZTogJzEnLFxuICAgIGNvbnRlbnQ6ICdUYWIgMSBjb250ZW50JyxcbiAgfSxcbiAge1xuICAgIHRpdGxlOiAnVGFiIDInLFxuICAgIG5hbWU6ICcyJyxcbiAgICBjb250ZW50OiAnVGFiIDIgY29udGVudCcsXG4gIH0sXG5dKVxuXG5jb25zdCBoYW5kbGVUYWJzRWRpdCA9ICh0YXJnZXROYW1lOiBzdHJpbmcsIGFjdGlvbjogJ3JlbW92ZScgfCAnYWRkJykgPT4ge1xuICBpZiAoYWN0aW9uID09PSAnYWRkJykge1xuICAgIGNvbnN0IG5ld1RhYk5hbWUgPSBgJHsrK3RhYkluZGV4fWBcbiAgICBlZGl0YWJsZVRhYnMudmFsdWUucHVzaCh7XG4gICAgICB0aXRsZTogJ05ldyBUYWInLFxuICAgICAgbmFtZTogbmV3VGFiTmFtZSxcbiAgICAgIGNvbnRlbnQ6ICdOZXcgVGFiIGNvbnRlbnQnLFxuICAgIH0pXG4gICAgZWRpdGFibGVUYWJzVmFsdWUudmFsdWUgPSBuZXdUYWJOYW1lXG4gIH0gZWxzZSBpZiAoYWN0aW9uID09PSAncmVtb3ZlJykge1xuICAgIGNvbnN0IHRhYnMgPSBlZGl0YWJsZVRhYnMudmFsdWVcbiAgICBsZXQgYWN0aXZlTmFtZSA9IGVkaXRhYmxlVGFic1ZhbHVlLnZhbHVlXG4gICAgaWYgKGFjdGl2ZU5hbWUgPT09IHRhcmdldE5hbWUpIHtcbiAgICAgIHRhYnMuZm9yRWFjaCgodGFiLCBpbmRleCkgPT4ge1xuICAgICAgICBpZiAodGFiLm5hbWUgPT09IHRhcmdldE5hbWUpIHtcbiAgICAgICAgICBjb25zdCBuZXh0VGFiID0gdGFic1tpbmRleCArIDFdIHx8IHRhYnNbaW5kZXggLSAxXVxuICAgICAgICAgIGlmIChuZXh0VGFiKSB7XG4gICAgICAgICAgICBhY3RpdmVOYW1lID0gbmV4dFRhYi5uYW1lXG4gICAgICAgICAgfVxuICAgICAgICB9XG4gICAgICB9KVxuICAgIH1cblxuICAgIGVkaXRhYmxlVGFic1ZhbHVlLnZhbHVlID0gYWN0aXZlTmFtZVxuICAgIGVkaXRhYmxlVGFicy52YWx1ZSA9IHRhYnMuZmlsdGVyKCh0YWIpID0+IHRhYi5uYW1lICE9PSB0YXJnZXROYW1lKVxuICB9XG59XG48L3NjcmlwdD5cbjxzdHlsZT5cbi5kZW1vLXRhYnMgPiAuZWwtdGFic19fY29udGVudCB7XG4gIHBhZGRpbmc6IDMycHg7XG4gIGNvbG9yOiAjNmI3NzhjO1xuICBmb250LXNpemU6IDMycHg7XG4gIGZvbnQtd2VpZ2h0OiA2MDA7XG59XG48L3N0eWxlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)

### Steps to reproduce

点击链接就可以看到，最右侧点击加号的图标颜色和左边tab的字体颜色不一样，应该是缩放导致了图标模糊。
无论tab改什么颜色都是一样的效果。
浏览器放大到300%两边颜色才一样

### What is Expected?

加号的图标颜色和左边tab的字体颜色一样

### What is actually happening?

加号的图标颜色和左边tab的字体颜色不一样

### Additional comments

_(empty)_

### PR DESCRIPTION
**Pull request by kooriookami**
closed #12028 #12130

Before
before

After
after

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by chenxch — APPROVED** [review]

File: packages/theme-chalk/src/tabs.scss [hunk 1]
```diff
@@ -392,11 +401,15 @@
...
+    }
+
+    .#{$namespace}-tabs__item.is-left {
+      justify-content: flex-end;
     }
-    .#{$namespace}-tabs__item.is-left,
+
     .#{$namespace}-tabs__item.is-right {
-      display: block;
+      justify-content: flex-start;
```
Comment (btea) [inline packages/theme-chalk/src/tabs.scss:412]: Are there compatibility issues with `justify-content`?

Comment (kooriookami) [reply packages/theme-chalk/src/tabs.scss:412]: Never mind. We already use these properties in other compoents.
image

Comment (chenxch) [reply packages/theme-chalk/src/tabs.scss:412]: ok.
/cc @btea

Comment (btea) [reply packages/theme-chalk/src/tabs.scss:412]: Ok.

**Review by btea — APPROVED** [review]

**Review by ryuhangyeong — APPROVED** [review]

**Review by emojiiii — APPROVED** [review]

**Pull request merged**